src/Altinn.AccessManagement.Persistence/DelegationMetadataRepo.cs

```diff
@@ -199,14 +199,28 @@ public async Task<DelegationChange> InsertDelegation(ResourceAttributeMatchType
 
             return await InsertResourceRegistryDelegation(delegationChange, cancellationToken);
         }
-
+        private string TranslateChangeType(DelegationChangeType type)
```

Comment:
Må nesten fikse disse "Closing brace should be followed by blank line" så build and test kjører